backend/server.py

```
@@ -3442,6 +3478,26 @@ def _record_priority(r: dict) -> int:
+    ALLOW_PAID_BRAIN=true overrides it.
+    """
+    from brain_policy import allow_paid_brain as _env_allow_paid
+    # Env var takes precedence over DB (operator kill-switch).
+    if _env_allow_paid():
+        return {"allow_paid": True, "surfaces": {}}
+    try:
+        doc = await get_db().providers.find_one({"provider_id": "provider_policy"})
+        if doc:
+            return {"allow_paid": bool(doc.get("allow_paid", False)), "surfaces": {}}
+    except Exception:
+        pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)